sdk/identity/azure-identity/tests/managed-identity-live/service-fabric/service_fabric.md

```diff
@@ -0,0 +1,211 @@
+# Testing azure-identity in Service Fabric
+
+Setup for a Service Fabric cluster and two apps, used for testing managed identity using Azure.Identity.
+
+The `sfmitestsystem` and `sfmitestuser` directories contain mock applications that use Azure.Identity's `ServiceFabricCredential` to request and verify Key Vault access tokens. The former application uses a system-assigned managed identity to do so, and the latter application uses a user-assigned managed identity.
+
+> **Note:** The code run by the applications comes from the `Dockerfile` in each, so adapting this sample to another language can be done by editing the Docker image each application uses. No other configuration changes are necessary if using Linux containers. To use Windows containers, you would need to edit the cluster configuration to run on Windows virtual machines (an example of a Windows configuration can be found [here](https://github.com/Azure/azure-quickstart-templates/tree/master/service-fabric-secure-cluster-5-node-1-nodetype)).
```

Comment:
This is vestigial now that the new Dockerfiles are gone. I'd probably just delete the paragraph because the goal here is to walk someone through a manual test.